python/docs/proton.handlers.rst

```diff
@@ -0,0 +1,106 @@
+##########################
+Module ``proton.handlers``
+##########################
+
+.. currentmodule:: proton.handlers
+
+Module Summary
+##############
+
+|
+
+.. autosummary::
+
+    MessagingHandler
+    EndpointStateHandler
```

Comment:
As above I don't think we should document anything other than MessagingHandler & after consideration TransactionHandler/TransactionClientHandler.

Reply:
Bear in mind this is because MessagingHandler actually aggregates all the other functionality anyway because of the way the event dispatch works. So all the default functionality is that of MessagingHandler.

Reply:
I went through this list with @ssorj, but if he agrees, we should remove it. Better now than after we release this. It is better to add bits than to have to retract/deprecate them later.